src/webapp/features/rabble_logo/rabbleLogo.tsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import { useEffect } from "react";
+import styles from './rabbleLogo.module.css';
+
+const RabbleLogo = () => {
+  // this just display:none's the container after a countdown to diminish the chance of weird interactions with it hiding in the z-index.
+  useEffect(() => {
+    const disappear = setTimeout(() => {
+      let logoContainer = document.getElementById("disappear");
+      if (logoContainer) logoContainer.style.display = "none";
+    }, 7000);
+    return () => {
+      clearTimeout(disappear);
+    }
+  }, [])
+
+    return (
+      <>
+    <div className={styles.rabbleLogo__container} id={"disappear"}>
+      <div className={styles.flex}>
+        <div className={styles.cube}>
+          <div className={`${styles.wall} ${styles.front}`}>
```

Comment:
Ooh i bet we can eliminate some of the repetition from this component by making a <Cube> component and passing it the side-letters as props!